app/javascript/controllers/leaflet_viewer_controller.js

```diff
@@ -197,6 +197,7 @@ export default class LeafletViewerController extends Controller {
     if (protocol == "Wms") return wmsLayer(url, options);
     if (protocol == "Wmts") return await wmtsLayer(url, options);
     if (protocol == "Xyz") return tileLayer(url, options);
+    if (!protocol && this.bounds instanceof L.LatLngBounds) return L.polygon([this.bounds.getSouthWest(), this.bounds.getSouthEast(), this.bounds.getNorthEast(), this.bounds.getNorthWest()]);
```

Comment:
isn't this exactly what `this.addBoundsOverlay()` does?

Reply:
imo this should go in `loadMap()` or somewhere else and just call `this.addBoundsOverlay()`, since this function is about adding actual data to the map

Reply:
I did not see that. Yes, so it looks like we aren't setting drawInitialBoundsValue, I will fix this.